<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog1</class>
 <widget class="QDialog" name="MyDialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>981</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>点右下角的↓可以切换下一句话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>271</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>这里可以放人物1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NexSenBtn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>630</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>271</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>这里可以放人物2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnC1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>这里是选项A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnC2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>这里是选项B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnC3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>这里是选项C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
